src/components/common/YourCompany/YourCompany.vue

```diff
@@ -209,43 +209,28 @@
       />
     </div>
 
-    <v-divider class="mx-4 my-1" />
-
     <!--- Association Type (coop only) -->
     <div v-if="isEntityTypeCP"
         id="association-type-section"
         class="section-container"
         :class="{'invalid-section': invalidAssociationTypeSection}"
     >
+      <v-divider class="mx-4 my-1" />
         <AssociationType
```

Comment:
This code block should line up with the divider above.

Reply:
Done.